app/controllers/links/pages_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Links::PagesController < PagesController
   def index
-    pages  = Comfy::Cms::Page.includes(:site).published.filter(params.slice(:category))
+    pages  = Comfy::Cms::Page.includes(:site).where('state != ?', 'unpublished').filter(params.slice(:category))
```

Comment:
Could we do the same thing with a `scope` in [`Comfy::Cms::Page`](https://github.com/moneyadviceservice/cms/blob/master/app/models/comfy/cms/page.rb) ?

```ruby
class Comfy::Cms::Page < ActiveRecord::Base
  scope :allthethings, -> { where.not(state: 'unpublished') }
end

def index
  pages = Comfy::Cms::Page.includes(:site).allthethings.filter(params.slice(:category)
end
```

or maybe use the existing `unpublished` scope and try something like:

```ruby
def index
  pages = Comfy::Cms::Page.includes(:site).where.not.published.filter(params.slice(:category)
end
```